packages/swap/src/routes/quote.ts

```diff
@@ -203,7 +233,11 @@ const quoteRouter = (io: Server) => {
         });
 
         if (result.success) {
-          res.json({ ...result.data.response, quoteType: undefined });
+          res.json({
+            ...result.data.response,
+            quoteType: undefined,
+            boostedQuote: isLocalnet() ? await getBoostedPoolQuoteResult(query) : undefined,
```

Comment:
think this will add the `boostedQuote` if we dont use the quoter?

Reply:
might make sense to move the `isLocalnet` check into `getBoostedPoolQuoteResult`, then we can call it multiple times

Reply:
Yeah, as i explained above, we will replace the current quoter with a simple RPC call so we decided to not spend too much time trying to make this work with the current quoter too.

I also kinda feel like the `isLocalnet` check here shouldn't be a responsibility of the `getBoostedPoolQuoteResult` method but instead the quote endpoint handler.

Reply:
Does that maybe provide more clearance/context?

Reply:
okay not a strong opinion, its fine for me both ways 🙂